test_runner/regress/test_tenants.py

```diff
@@ -397,3 +397,34 @@ def test_pageserver_with_empty_tenants(
     assert (
         tenant_broken_count == 1
     ), f"Tenant {tenant_without_timelines_dir} should have metric as broken"
+
+
+# Check that empty tenants work with or without the remote storage
+@pytest.mark.parametrize(
+    "remote_storage_kind", available_remote_storages() + [RemoteStorageKind.NOOP]
+)
+def test_pageserver_create_tenants_fail(
+    neon_env_builder: NeonEnvBuilder, remote_storage_kind: RemoteStorageKind
+):
+    neon_env_builder.enable_remote_storage(
+        remote_storage_kind=remote_storage_kind,
+        test_name="test_pageserver_create_tenants_fail",
+    )
+
+    env = neon_env_builder.init_start()
+
+    env.pageserver.allowed_errors.append(".*tenant-create-fail.*")
+    env.pageserver.allowed_errors.append(".*Tenant is already in Broken state.*")
+    env.pageserver.allowed_errors.append(".*could not load tenant.*")
+
+    client = env.pageserver.http_client()
+    client.configure_failpoints(("tenant-create-fail", "return"))
+    tenant_id = TenantId("deadbeefdeadbeefdeadbeefdeadbeef")
+    try:
+        client.tenant_create(tenant_id)
+    except Exception as e:
+        exception_string = str(e)
+        assert "tenant-create-fail" in exception_string, "should reach failpoint"
```

Comment:
You can use `pytest.raises` here, in one of the tests we have similar construction:

```suggestion
    with pytest.raises(PageserverApiException, match=f"failpoint: tenant-create-fail"):
        client.tenant_create(tenant_id)
```